README.md

```diff
@@ -39,6 +39,55 @@ library = foreman-ansible-modules/modules
 
 Now your playbooks and roles should have access to the modules contained in the repository for use, testing or development of new modules.
 
+## How to debug this repository
+
+Clone both ansible and this project
+
+```sh
+git clone https://github.com/ansible/ansible.git
+git clone https://github.com/theforeman/foreman-ansible-modules.git
```

Comment:
No, don't re-clone. Just your local working copy. Perhaps we should remove this line from the readme?